src/devices/MemoryLCD/Extends/LSxxxB7DHxxExtends.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Drawing;
+using Iot.Device.MemoryLcd;
+
+namespace MemoryLcd.Extends
+{
+    /// <summary>
+    /// Extend methods for LSxxxB7DHxx
+    /// </summary>
+    public static class LSxxxB7DHxxExtends
```

Comment:
We usually name these types of classes Extensions instead of Extends, so in this case this should be named `LSxxxB7DHxxExtensions`